src/RpcServer/Session.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2015-2022 The Neo Project.
+//
+// The Neo.Network.RPC is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Iterators;
+using Neo.SmartContract.Native;
+using System;
+using System.Collections.Generic;
+
+namespace Neo.Plugins
+{
+    class Session : IDisposable
+    {
+        public static readonly TimeSpan ExpirationTime = TimeSpan.FromMinutes(5);
```

Comment:
ExpirationTime probably should be configurable. 5 minutes seems VERY long and subject to DOS

Reply:
Done.